app/controllers/ad_renewal_letters_exports_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class AdRenewalLettersExportsController < ApplicationController
+  def index
+    authorize! :read, WasteExemptionsEngine::AdRenewalLettersExport
+
+    @ad_renewal_letters_exports = WasteExemptionsEngine::AdRenewalLettersExport.all
+  end
+
+  def update
+    @ad_renewal_letters_exports = WasteExemptionsEngine::AdRenewalLettersExport.find(params[:id])
```

Comment:
Is the reason `@ad_renewal_letters_exports` is plural in `update()` so that it works with the strong params / `ad_renewal_letters_exports_attributes` function?

If it isn't, I'd just ask if we could make it `@ad_renewal_letters_export` in as we're only dealing with the one instance here.

If it is, don't bother about it. This is just a NIT 😁

Reply:
Oh!! This is sop true! I have added too many `s`s XD